scripts/networks/deploymentFunctions.ts

```diff
@@ -1833,3 +1835,45 @@ export const prechecks: IDeploymentFunction = async function (params: IDeploymen
   }
   return {}
 }
+
+export const extractTokenConfig = async function (params: IDeploymentParams): Promise<ITokenConfig[]> {
+  const getContract = await getContractFromSection(params.existing)
+  const LR_old = await getContract<Types.LiquidityReceiverV1>(Sections.LiquidityReceiverOld, "LiquidityReceiverV1");
+  const LR_new = await getContract<Types.LiquidityReceiver>(Sections.LiquidityReceiverNew, "LiquidityReceiver");
```

Comment:
Just for consideration: I wonder if it wouldn't be better to use e.g. `LRV1` and `LRLatest` names. Are we confident that there's no chance for `LiquidityReceiverV3` in the future?

Reply:
Ha ha, I left a comment in the UI (still to be committed) telling anyone reading the code to put those questions to bed. It's a weird situation where there are only 2 LRs. What I'm doing in the UI is to refer to LiquidityReceiverV3 as a shorthand for "Liquidity Receiver for PyroTokens V3". The old, new business is confined to the deployment scripts. Nonetheless the issue is contained in the dev-env app and doesn't leak into the broader UI. At some point I think the universal naming scheme of association with the Pyrotokens version is the most sane.
It feels a little like how windows went from 8 to 10, though.